prprocessor/config/repos.yaml

```diff
@@ -74,6 +74,10 @@ theforeman/foreman_remote_execution:
   redmine: foreman_remote_execution
   redmine_required: true
   redmine_version_prefix: "foreman_remote_execution "
+theforeman/katello-pull-transport-migrate:
+  redmine: foreman_remote_execution
+  redmine_required: true
+  redmine_version_prefix: "foreman_remote_execution "
```

Comment:
I don't think this is correct. This version prefix is what you can see in https://projects.theforeman.org/projects/foreman_remote_execution/roadmap. This would imply it adds a Fixed in for that (sub)project. I'd expect it to have its own prefix, even if you don't actually track the versions. Otherwise it'll be assigned to some random version.